.gitattributes

```diff
@@ -0,0 +1 @@
+* text eol=lf
```

Comment:
Adding `.gitattributes` per information in https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings - it might convince Windows to stick with LF.

Reply:
This will override line endings in any file - can we limit it to just `*.enso`, please. We do have CSV and txt files with deliberate Windows endings.

Reply:
Treating all files in the repo as `text` somehow feels wrong. Why don't we limit it to the enso files?

Reply:
We aren't supposed to have any binaries in the repository. Pictures are downloaded via other means. Moreover this `text eol=lf` setting is supposed to be equivalent to the setting used on Windows so far.